<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>genral</class>
 <widget class="QMainWindow" name="genral">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>560</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>16</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0" colspan="2">
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <property name="spacing">
       <number>5</number>
      </property>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string/>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="3" column="0" alignment="Qt::AlignHCenter">
            <widget class="QLabel" name="label_7">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>16</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Valve Setting:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="3">
            <widget class="QGroupBox" name="groupBox_4">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>200</height>
              </size>
             </property>
             <property name="title">
              <string/>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="label_8">
                <property name="maximumSize">
                 <size>
                  <width>47</width>
                  <height>38</height>
                 </size>
                </property>
                <property name="baseSize">
                 <size>
                  <width>20</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="pixmap">
                 <pixmap>icons/LOA3.png</pixmap>
                </property>
                <property name="scaledContents">
                 <bool>true</bool>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="wordWrap">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item alignment="Qt::AlignHCenter">
               <widget class="QLabel" name="title_2">
                <property name="font">
                 <font>
                  <pointsize>20</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="cursor">
                 <cursorShape>IBeamCursor</cursorShape>
                </property>
                <property name="focusPolicy">
                 <enum>Qt::StrongFocus</enum>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(139, 202, 97)</string>
                </property>
                <property name="text">
                 <string>Pressure Control</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="5" column="2">
            <widget class="QPushButton" name="plotButton">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>16</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Plot</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0" alignment="Qt::AlignHCenter">
            <widget class="QLabel" name="label_2">
             <property name="minimumSize">
              <size>
               <width>170</width>
               <height>0</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>16</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>P Actual (Bars):</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="3" column="1" alignment="Qt::AlignHCenter">
            <widget class="QLabel" name="label_valve_status">
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>50</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>16</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Open/Closed/PID</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1" alignment="Qt::AlignHCenter">
            <widget class="QLabel" name="inlet_valve_label">
             <property name="font">
              <font>
               <pointsize>16</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>55</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0" alignment="Qt::AlignHCenter">
            <widget class="QLabel" name="label_9">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>25</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>16</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Valve Opening (%):</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1" alignment="Qt::AlignHCenter">
            <widget class="QLabel" name="measure">
             <property name="minimumSize">
              <size>
               <width>90</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>90</width>
               <height>60</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>30</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>1.05</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>140</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>500</height>
         </size>
        </property>
        <property name="title">
         <string>PID settings</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="i_gain_box">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>50</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QDoubleSpinBox" name="d_gain_box">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>50</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QDoubleSpinBox" name="p_gain_box">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>50</height>
            </size>
           </property>
           <property name="decimals">
            <number>0</number>
           </property>
           <property name="maximum">
            <double>10000.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="2" alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>D</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>P</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>I</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QPushButton" name="set_pid_button">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>500</height>
            </size>
           </property>
           <property name="text">
            <string>SET PID</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox_5">
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="3" alignment="Qt::AlignHCenter">
          <widget class="EnterSpinBox" name="setpoint">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>200</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>16</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="maximum">
            <double>100.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="3" column="0" alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label_6">
           <property name="font">
            <font>
             <pointsize>16</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Valve Control:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2" colspan="2" alignment="Qt::AlignHCenter">
          <widget class="QGroupBox" name="groupBox_6">
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>300</height>
            </size>
           </property>
           <property name="title">
            <string/>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="force_open_button">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>50</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>50</height>
               </size>
              </property>
              <property name="text">
               <string>Open</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="openButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>50</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>50</height>
               </size>
              </property>
              <property name="text">
               <string>PID</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="closeButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>50</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>50</height>
               </size>
              </property>
              <property name="text">
               <string>Close</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="2" alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>16</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>P Set (Bars)</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>499</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EnterSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>flowControl</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
